config/i18n.ts

```diff
@@ -213,6 +213,11 @@ const locales: LocaleObjectData[] = [
     file: 'tl-PH.json',
     name: 'Tagalog',
   },
+  {
+    code: 'vi-VN',
+    file: 'vi-VN.json',
+    name: 'Vietnamese',
```

Comment:
shouldn't `Vitenamese` also be translated here?

Reply:
@mastoduy `Tiếng Việt`

Reply:
update to use first one:

![imagen](https://github.com/elk-zone/elk/assets/6311119/fdcf1b2e-5a2b-4a92-a235-f77c51114fe5)